src/components/autocompleter/mock-handlers.js

```diff
@@ -0,0 +1,71 @@
+import { rest } from 'msw';
+import mockConstants from '../../mocks/mock-constants.js';
+
+const { API_ROOT } = mockConstants;
+
+const models = [
+  {
+    id: '0',
```

Comment:
I'd recommend using random IDs. We used to have a bug that assumed the response object for the models was an array instead of a lookup. 

I guess at the very least, don't start with `0`. That makes it look very "array-y"

Reply:
Probably another thing we should standardize on moving forward. Also seems like it would be made moot in the bright future of importing brainstem factories.